R/preview.R

```diff
@@ -7,7 +7,10 @@ form_caption <- function(long = NULL, short = NULL, label = NULL) {
   if(is.character(label)) {
     label <- paste0(" \\", label)
   }
-  paste0("\\caption[",short,"]{",long,label,"}")
+  if(is.character(short)) {
+    short <- paste0("[", short, "]")
+  }
```

Comment:
When `short` is `NULL`, the short name becomes `[]` and nothing shows up in toc. Refactoring this to only bracket when there is something to bracket.